From 62ce7cbc96a52723e11adfda848211e03f6733ba Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Wed, 3 May 2017 14:04:17 +0200
Subject: [PATCH 3/3] System libs

---
 CMakeLists.txt | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bc5cf85..07e4c41 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,9 +46,7 @@ if (ENABLE_OPENMP)
     endif()
 endif()
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR} libs)
-
-add_library(clipper STATIC libs/clipper/clipper.cpp)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 set(engine_SRCS # Except main.cpp.
     src/bridge.cpp
@@ -136,7 +134,7 @@ endif ()
 
 # Compiling CuraEngine itself.
 add_library(_CuraEngine ${engine_SRCS} ${engine_PB_SRCS}) #First compile all of CuraEngine as library, allowing this to be re-used for tests.
-target_link_libraries(_CuraEngine clipper)
+target_link_libraries(_CuraEngine polyclipping)
 if (ENABLE_ARCUS)
     target_link_libraries(_CuraEngine Arcus)
 endif ()
-- 
2.11.0

