--- CuraEngine-4.13.2/CMakeLists.txt.orig	2024-09-26 21:13:45.678247207 +0200
+++ CuraEngine-4.13.2/CMakeLists.txt	2024-09-26 21:14:33.027990691 +0200
@@ -28,8 +28,6 @@ if(ENABLE_ARCUS)
     endif()
 endif()
 
-#For reading image files.
-find_package(Stb REQUIRED)
 include_directories(${Stb_INCLUDE_DIRS})
 
 option(USE_SYSTEM_LIBS "Use the system libraries if available" OFF)
@@ -277,9 +275,6 @@ endif ()
 # Compiling CuraEngine itself.
 add_library(_CuraEngine STATIC ${engine_SRCS} ${engine_PB_SRCS}) #First compile all of CuraEngine as library, allowing this to be re-used for tests.
 
-if (CuraEngine_Download_Stb)
-    add_dependencies(_CuraEngine stb)
-endif()
 if(USE_SYSTEM_LIBS)
     target_link_libraries(_CuraEngine ${Polyclipping_LIBRARIES})
 else()
